### PR TITLE
Initial import

### PR DESCRIPTION
**Pull request by astashov**

### CODE REVIEW

**Comment by astashov** [comment]
@danschultz PTAL

File: lib/utils.dart [hunk 1]
```diff
@@ -0,0 +1,14 @@
+library tasks.utils;
+
+import 'dart:async';
+import 'package:tasks/tasks.dart';
+
+Future pmap(Iterable items, block(item), {int concurrencyCount: 1}) async {
```
Comment (danschultz) [inline lib/utils.dart]: This verbiage is a bit obscure.

**Comment by danschultz** [comment]
LGTM, but would recommend renaming `pmap` to something that's a verb and also more descriptive.

File: lib/utils.dart (hunk 1 above)
Comment (astashov) [inline lib/utils.dart]: Seems to be pretty common name for a parallel version of map:

http://docs.julialang.org/en/latest/stdlib/parallel/?highlight=pmap#Base.pmap
http://lparallel.org/pmap-family/
https://github.com/bruceadams/pmap

etc

**Pull request merged**